src/components/Pagination/PaginationService.js

```diff
@@ -0,0 +1,79 @@
+import {
+  ceil,
+  compose,
+  divide,
+  isEmpty,
+  first,
+  last,
+  gt,
+  add,
+  subtract,
+  lt,
+  __,
+  sortBy,
+  identity,
+  reverse
+} from 'lodash/fp';
+
+export const calculatePages = (totalItems, perPage) =>
+  compose(ceil, divide(totalItems))(perPage);
+
+export const createEmptyArrayFromNumber = length => Array.from({ length });
+
+const availableSortsForPrevious = (page, totalPages) => {
+  if (page === totalPages) {
+    return 3;
+  }
+
+  return page + 1 === totalPages ? 2 : 1;
+};
+
+const availableSortsForNext = page => {
```

Comment:
This looks exactly the same as `availableSortsForPrevious` but with a fixed parameter on `totalPages`